api/src/main/java/org/openmrs/module/aijar/dataintegrity/InvalidTBEncounters.java

```diff
@@ -67,14 +67,14 @@
 			Patient patient = obs.getEncounter().getPatient();
 			
 			RuleResult<Patient> ruleResult = new RuleResult<>();
-			ruleResult.setActionUrl("coreapps/patientdashboard/patientDashboard.page?patientId=" + patient.getUuid());
-			ruleResult.setNotes("The District TB Number " + getTbNumber(patient, obs.getEncounter()) + " is used by another patient");
+			ruleResult.setActionUrl("htmlformentryui/htmlform/editHtmlFormWithStandardUi.page?patientId=" + patient.getUuid() + "&encounterId=" + obs.getEncounter().getId());
+			ruleResult.setNotes("The " + identifierTitle + " is used by another patient");
```

Comment:
@ssmusoke Sorrt,  I actually thought that's what you had asked me to do here. 
![image](https://user-images.githubusercontent.com/1963527/28995633-b925fba8-79f6-11e7-916f-3d68872bd16d.png) 

I have addressed it.